dashboard/db/migrate/20210121195951_create_foorm_libraries.rb

```diff
@@ -0,0 +1,13 @@
+class CreateFoormLibraries < ActiveRecord::Migration[5.2]
+  def change
+    create_table :foorm_libraries do |t|
+      t.string :name, null: false
+      t.integer :version, null: false
+      t.boolean :published, default: true, null: false
```

Comment:
curious about using `true` as the default here. i would expect libraries to start out unpuplished, no? imo might make more sense to set this to `true` during seeding so that once we support creating new libraries (if that's ever going to be a thing), new libraries will start out unpubplished. but maybe i don't have proper context on this.

Reply:
I can give context on this. When we started adding the publish option for forms/library questions I set the default to true because none of the forms had a pre-set published state yet, and we aren't using the published state currently. I think it could be worth changing the default in the future when we have a more formalized use of published.

Reply:
I think that's a good call -- looks like the seeding logic defaults to true, so we're already doing what you suggested :). I think I'll just skip having a db-level default, and rely on places where we save libraries (currently, only during seeding) to decide whether it should be default published or not.

Reply:
@molly-moen do you think it's reasonable to not have a DB-level default since the seeding logic sets published to true if it's missing?

Reply:
my concern was that is while we will almost always create libraries by seeding, I didn't want us to get into a case where the published state was nil if for some reason that process changes. It is an edge case though, and not a major issue if it does end up nil, so I will leave it up to you.